regression/quokka-tests.ini

```diff
@@ -176,3 +176,20 @@ doVis = 1
 visVar = z-BField
 testSrcTree = .
 ignore_return_code = 0
+
+[Turbulence-GPU]
+buildDir = .
+target = Turbulence
+inputFile = inputs/Turbulence.in
```

Comment:
You can create a separate input file for the nightly regression tests. This is done for a few other problems. It could be, for example `inputs/Turbulence_regression.in`.